docs/spec/lsm/adr-002.md

```diff
@@ -0,0 +1,362 @@
+# ADR-002: Liquid Staking Module
+
+## Abstract
+
+The LSM is designed to safely and efficiently facilitate the adoption of liquid staking.
+
+The LSM mitigates liquid staking risks by limiting the total amount of tokens that can be liquid staked to 25% of all staked tokens. 
+
+As additional risk-mitigation features, the LSM introduces a requirement that validators self-bond tokens to be eligible for delegations from liquid staking providers, and that the portion of their liquid staked shares must not exceed 50% of their total shares.
+
+## Context 
+
+Liquid proof of stake systems  exacerbate the risk that a single entity - the liquid staking provider - amasses more than ⅓ the total staked supply on a given chain, giving it the power to halt that chain’s block production or censor transactions and proposals.
+
+Liquid proof of stake may also exacerbates the principal agent risk that exists at the heart of the delegated proof of stake system. The core of the problem is that validators do not actually own the stake that is delegated to them. This leaves the open to perverse incentives to attack the consensus system. Cosmos introduced the idea of min self bond in the staking. This creates a minimum amount of stake the must be bonded by the validators operator key. This feature has very little effect on the behavior of delegates.
+
+
+## Proposal:
+
+
+### Limiting liquid staking
+
+
+The LSM would limit the percentage of liquid staked tokens by all liquid staking providers to 25% of the total supply of staked tokens. For example, if 100M tokens were currently staked, and if the LSM were installed today then the total liquid staked supply would be limited to a maximum of 25M tokens.
+
+This is a key safety feature, as it would prevent liquid staking providers from collectively controlling more than ⅓ of the total staked token supply, which is the threshold at which a group of bad actors could halt block production.
+
+Additionally, a separate cap is enforced on each validator's portion of liquid staked shares. Once 50% of shares are liquid, the validator is unable to accept additional liquid stakes.
+
+Technically speaking, this cap on liquid staked tokens is enforced by limiting the total number of tokens that can be staked via interchain accounts plus the number of tokens that can be tokenized using LSM. Once this joint cap is reached, the LSM prevents interchain accounts from staking any more tokens and prevents tokenization of delegations using LSM.
+
+
+### Validator self-bond
```

Comment:
"self-bond" is not accurate as any delegator can choose to do that, it doesn't have to be the validator